### ISSUE TITLE
Lots of Errors!!! Plz help!!

### ISSUE THREAD

**Issue by GabrielTK** [maierfelix/POGOserver#229]
Good Evening!
I've setup this in docker, but now I'm getting another error:
```
[Console] https://github.com/maierfelix/POGOServer
[Console] Booting Server v0.5.9
[Console] Downloaded assets are valid! Proceeding..
[Console] Creating table gym
[Console] Creating table users
[Console] Creating table spawn_points
[Console] Creating table pokestop
[Console] Creating table owned_pkmn
[Console] MySQL connection established
[Console] Server listening at 172.17.0.6:3000
[Console] Invalid authentication token! Kicking..
[Console] Removed null
[Console] Invalid authentication token! Kicking..
[Console] Removed null
```
Can anyone help me please??
**NOTE:** `docker-machine` doesn't work, because of Hyper-V.

**Comment by GabrielTK** [comment]
Ok, now another Error:
```
            ______ _____ _____ _____
            | ___ \  _  |  __ \  _  |
            | |_/ / | | | |  \/ | | | ___  ___ _ ____   _____ _ __
            |  __/| | | | | __| | | |/ __|/ _ \ '__\ \ / / _ \ '__|
            | |   \ \_/ / |_\ \ \_/ /\__ \  __/ |   \ V /  __/ |
            \_|    \___/ \____/\___/ |___/\___|_|    \_/ \___|_|

[Console] https://github.com/maierfelix/POGOServer
[Console] Booting Server v0.5.9
[Console] Downloaded assets are valid! Proceeding..
[Console] MySQL connection established
[Console] Server listening at 172.17.0.2:3000
[Console] Unhandled exception occurred:
[Console] TypeError: Cannot read property 'contents' of undefined
[Console] TypeError: Cannot read property 'contents' of undefined
    at GameServer.authenticatePlayer (/POGOserver/src/request.js:190:31)
    at GameServer.onRequest (/POGOserver/src/request.js:97:10)
    at GameServer.processRpcRequest (/POGOserver/src/request.js:84:10)
    at GameServer.routeRequest (/POGOserver/src/request.js:28:12)
    at IncomingMessage.<anonymous> (/POGOserver/src/http.js:22:12)
    at emitNone (events.js:86:13)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickDomainCallback (internal/process/next_tick.js:122:9)
[Console] The server has crashed!
```

Note: This is my Fiddler Script for redirection, as **Xposed Pokemon Go asks me to update** (I'm using 0.35.0):
```
static function OnBeforeRequest(oSession: Session) {
        // Sample Rule: Color ASPX requests in RED
        // if (oSession.uriContains(".aspx")) {	oSession["ui-color"] = "red";	}
           if (oSession.uriContains("pgorelease.nianticlabs.com/plfe/rpc")) {
            oSession["ui-color"] = "blue";
            oSession.host = "localhost";
            oSession.port=3000;
            oSession.oRequest.headers.UriScheme = "http";

		}
  }
```

**Comment by nliakm** [comment]
because of the anniversary update , i cant use any emulators while hyper-v is enabled. so i can't test it.
but does it help, if you use 0.33.0 and change the line in cfg.js to this one?
`MINIMUM_CLIENT_VERSION: "0.33.0",`

**Comment by GabrielTK** [comment]
Ok, I'll try. But why don't you install the latest **Docker for Windows**??? It runs over Hyper-V

**Comment by nliakm** [comment]
thats the problem. docker needs hyper-v to run. but when i run bluestack or nox, i get a bsod (system service exception).
so when hyper-v is enabled, i can run my server, but not the emulators and when i disable hyper-v, i can run the emulators, but not my server...
but i will fix it later, i'll just run my server on an other machine.

**Comment by DracoMilesX** [comment]
You can install docker toolbox which installs docker over virtual box

**Comment by GabrielTK** [comment]
So, any idea?? Anyone??

**Comment by GabrielTK** [comment]
Solved at PR #238
